src/pybind/mgr/dashboard/frontend/src/app/core/navigation/notifications/notifications.component.scss

```diff
@@ -7,3 +7,13 @@
 .running:hover i {
   color: bd.$color-solid-white;
 }
+
+.dot {
```

Comment:
I would suggest to use this CSS style because drawing a border around the circle makes it more visible. I used 2px because GitHub is also using this border size.

GitHub:
![Bildschirmfoto vom 2020-07-17 12-09-25](https://user-images.githubusercontent.com/1897962/87778539-a6742580-c82b-11ea-8eec-403cab4eb304.png)

Before:
![Bildschirmfoto vom 2020-07-17 12-43-18](https://user-images.githubusercontent.com/1897962/87778555-abd17000-c82b-11ea-95bd-f255e880fd8a.png)

After:
![Bildschirmfoto vom 2020-07-17 12-49-21](https://user-images.githubusercontent.com/1897962/87778756-f5ba5600-c82b-11ea-8289-83f4567cc684.png)

Note, when hovering the mouse over the icon the notification indicator is not visible, without and with a border. I don't think we can do anything here.

```
a {
  .dot {
    background-color: bd.$color-blue;
    border-radius: 50%;
    height: 10px;
    position: absolute;
    right: 16px;
    top: 13px;
    width: 10px;
    border: 2px solid bd.$color-navbar-bg;
  }
  &:hover .dot {
    border-color: bd.$color-primary;
  }
}
```

Reply:
Thanks @votdev.! It looks better with the border. And while hovering, using a brighter color for the background-color may help in this regard(example red)..but am not sure If we can use a color except for blue.